### PR TITLE
docs - add info about new init --skip-register option

### PR DESCRIPTION
**Pull request by lisakowen**
docs changes for https://github.com/greenplum-db/pxf/pull/477

### CODE REVIEW

**Review by frankgh — APPROVED** [review]
Looks good to me! Thanks Lisa

**Review by dyozie — APPROVED** [review]

File: docs/content/ref/pxf.html.md.erb [hunk 1]
```diff
@@ -75,11 +75,14 @@ You can copy the extension files from a PXF installation on the host to the Gree
 
 ## <a id="options"></a>Options
 
-The `pxf init` command takes the following option:
+The `pxf init` command takes the following options:
 
 <dt>&#8211;y </dt>
 <dd>Do not prompt, use the default `$PXF_CONF` directory location if the environment variable is not set.</dd>
 
+<dt>&#8211;&#8211;skip-register </dt>
```
Comment (denalex) [inline docs/content/ref/pxf.html.md.erb:83]: this is a bit inconsistent `-y` and `--skip-register`. I believe `-s` will also work, should we just document the short one ?

Comment (frankgh) [reply docs/content/ref/pxf.html.md.erb:83]: well, we have `-d` and `--delete` for sync. I think it makes sense in that context

Comment (frankgh) [reply docs/content/ref/pxf.html.md.erb:83]: I prefer `--skip-register` since it's self explanatory

**Review by denalex — APPROVED** [review]

**Pull request merged**